dactyl-config.yml

```
@@ -4945,6 +4945,21 @@ pages:
         top_nav_grouping: 開発
         targets:
             - ja
+    -   name: XRPL Brand Kit
+        html: https://github.com/XRPLF/xrpl-dev-portal/raw/master/content/XRPL_Brand_Kit.zip
+        parent: resources.html
+        blurb: Review guidelines for using XRPL design assets.
+        top_nav_grouping: Development
+        targets:
+            - en
+
+    -   name: XRPLブランドキット
+        html: https://github.com/XRPLF/xrpl-dev-portal/raw/master/content/XRPL_Brand_Kit.zip
+        parent: resources.html
+        blurb: XRPLのデザインアセットのガイドラインを確認しましょう。
+        top_nav_grouping: Development
```

```suggestion
        top_nav_grouping: 開発
```

In `targets: ja`, "開発" is used for “Development"

https://github.com/XRPLF/xrpl-dev-portal/blob/db87dc69635b18c0d16bb8eab680a3b86258599b/dactyl-config.yml#L4788